src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -1,10 +1,10 @@
 package mate.academy.model;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
```

Comment:
Why did you switch to jakarta?

Reply:
Because jakarta is the newest API that replaced javax. If there are no special requirements in the technical specification of the application, it is better to use modern technologies for development. That's what I'm trying to do.